packages/ag-charts-community/src/chart/agChartV2.ts

```diff
@@ -458,67 +459,55 @@ function applyModules(chart: Chart, options: AgChartOptions) {
 function applySeries(chart: Chart, options: AgChartOptions) {
     const optSeries = options.series;
     if (!optSeries) {
-        return false;
+        return 'no-change';
```

Comment:
I'd be inclined to make these an enum

Reply:
I don't see the benefit of an enum for these cases?

Reply:
Discussed offline, I've added `SeriesChangeType` as an explicit type to cleanup use of `ReturnType`.